core/cautils/getter/loadpolicy.go

```diff
@@ -130,13 +131,18 @@ func (lp *LoadPolicy) GetControlsInputs(clusterName string) (map[string][]string
 	filePath := lp.filePath()
 	accountConfig := &armotypes.CustomerConfig{}
 	f, err := os.ReadFile(filePath)
+	fileName := filepath.Base(filePath)
 	if err != nil {
+		logger.L().Error(fmt.Sprintf("Error opening %s file, \"controls-config\" will be downloaded from ARMO management portal", fileName))
```

Comment:
The error should not be printed from within the function but in the caller function.

Reply:
Hi @dwertent, 
Now, it is printing from the caller function.